src/Game.js

```diff
@@ -0,0 +1,48 @@
+function Game() {
+  this._total_score = 0;
+  this._frame_score = 0;
+  this.is_a_strike = false;
+  this.is_a_spare = false;
+  this.frame_count = 0;
+};
+
+Game.prototype.total_score = function(roll1, roll2) {
+  this.add_to_score(roll1, roll2);
+  return this._total_score;
+};
+
+Game.prototype.frame_score = function(roll1, roll2) {
```

Comment:
When rolling a strike, I get a NaN when leaving the second roll empty. Could add a default parameter of 0 for roll 2?

Reply:
completely agree! I also wanted to have the second roll to adapt to the first one so you couldn't roll more than 10 in one frame.